tfx/types/standard_component_specs.py

```diff
@@ -100,7 +100,9 @@ class EvaluatorSpec(ComponentSpec):
 class ExampleValidatorSpec(ComponentSpec):
   """ExampleValidator component spec."""
 
-  PARAMETERS = {}
+  PARAMETERS = {
+      'exclude_splits': ExecutionParameter(type=List[Text], optional=True),
```

Comment:
it might need to be List[(str, Text)] or (List[str], List[Text]), please check

Reply:
both of these will fail the component tests